index.html

```diff
@@ -0,0 +1,175 @@
+<html>
+<head>
+		<meta charset="UTF-8" />
+		<title>Homework-VueJS-HTML</title>
+		<meta content="IE=edge" http-equiv="X-UA-Compatible" />
+		<meta content="width=device-width, initial-scale=1" name="viewport" />
+
+    <style>
+        body {
+            background-color: #ffeead;
+			font-size: 1.2em;
+            font-weight: bold;
+        }
+        .container {
+            display: grid;
+            grid-gap: 3px;
+            grid-template-columns: repeat(12, 1fr);
+            grid-template-rows: 50px auto 50px;
+            height: 100%;
+        }
+        .header {
+            grid-column: 2 / -1;
+            background-color: #fd726d;
+            display: flex;
+            padding: 20px;
+        }
+
+		.center_text {
+            color: #ffeead;
+            display:flex;
+            align-items: center; 
+            justify-content: center; 
+        }
+
+		.header > div:nth-child(-n+3) {
+			margin-right: 20px;
+        }
+       
+        .header > div:nth-child(4) {
+			margin-left: auto;
+        }
+
+        .logo {
+            grid-row: 1 / 2;
+            background-color: #98cdb5;
+        }
+
+        .sidebar {
+            grid-row: 2 / 3;
+            background-color: #20cb9b;
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            justify-content: space-between;
+            padding: 30px;
+        }
+
+        .sidebar div {
+            margin-top: 20px;
```

Comment:
Better add margin top to all elements, except first one
`.sidebar div:not(:first-child)`